configs/nextflow.hopper.config

```diff
@@ -8,16 +8,14 @@ singularity{
 params {
 	root = "/fs1/pipelines/JASEN"
 	// DATABASES //
+	amrfinderDb = "${root}/assets/amrfinder_db/2023-04-17.1"
```

Comment:
Could this be renamed to a generic name so the config doesn't have to be updated for every version

Reply:
`amrfinderDb = "${root}/assets/amrfinder_db/latest"`?